src/components/common/SpinnerPortal/Spinner.js

```diff
@@ -12,4 +12,16 @@ const SpinnerComponent = ({ isFullScreen = true, customClasses = '' }) => {
   );
 };
 
-export { SpinnerComponent };
+const DotFlashingSpinner = ({ customClasses = '', isShow }) => {
+  return (
+    <div className={`dot-flashing-wrapper ${customClasses}`}>
+      <div className="snippet" data-title=".dot-flashing">
+        <div className="stage">
+          {isShow && <div className="dot-flashing"></div>}
+        </div>
+      </div>
+    </div>
```

Comment:
if `isShow` is false, looks like we will be just showing blank divs in our dom. Would it make sense to move the condition at the top and display `null`?